pages/pipelines/managing_priorities.md

```diff
@@ -17,6 +17,28 @@ steps:
 
 Job priority is considered before jobs are dispatched to [agent queues](/docs/agent/v3/queues), so jobs with higher priority are assigned before jobs with lower priority, regardless of which has been longest in the queue. Priority only applies to command jobs, including plugin commands.
 
+## Prioritizing whole builds
+
+`priority` can be set as a top-level value, applying it to all steps in the pipeline which do not have their own `priority` set. This is useful when you may need an entire pipeline to be considered a higher priority than others:
+
+This may be
```

Comment:
Thanks again for this contribution @mcncl !

Just suggest re-writing the intro to match the format of the previous heading's intro...

```suggestion
The `priority` key can be set as a top-level value, which applies it to all steps in the pipeline that do not have their own `priority` key set. This is useful when an entire pipeline requires a higher priority than others. For example:
```